src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -214,14 +214,25 @@ bool clang_c_convertert::get_decl(const clang::Decl &decl, exprt &new_expr)
     const clang::TranslationUnitDecl &tu =
       static_cast<const clang::TranslationUnitDecl &>(decl);
 
+    msg.debug(fmt::format(
+      "TranslationUnit has a total of {} top-level declarations",
+      std::distance(std::cbegin(tu.decls()), std::cend(tu.decls()))));
+    static unsigned ctr = 0;
     for(auto const &decl : tu.decls())
     {
       // This is a global declaration (variable, function, struct, etc)
       // We don't need the exprt, it will be automatically added to the
       // context
       exprt dummy_decl;
+      if(ctr == 10)
```

Comment:
Why 10?

Reply:
Removed. 
Sorry this was for debugging purposes.